process/sync/metablock.go

```diff
@@ -412,17 +402,14 @@ func (boot *MetaBootstrap) getBlockBodyRequestingIfMissing(headerHandler data.He
 	return blockBody, nil
 }
 
-func (boot *MetaBootstrap) requestMiniBlocksFromHeaderWithNonceIfMissing(_ uint32, nonce uint64) {
+func (boot *MetaBootstrap) requestMiniBlocksFromHeaderWithNonceIfMissing(hash []byte, nonce uint64) {
```

Comment:
Now it is more clear that you can send headerHandler instead hash  and we can remove line 412, no need to access pool again.

Reply:
done.